spec/fuzzy-finder-spec.coffee

```diff
@@ -404,6 +404,9 @@ describe 'FuzzyFinder', ->
           dispatchCommand('toggle-buffer-finder')
 
       describe "when the active pane has an item for the selected path", ->
+        beforeEach ->
+          atom.config.set("fuzzy-finder.searchAllPanes", false)
```

Comment:
Since `false` is the default value for this config option, is this call to `atom.config.set` needed here? I'm wondering if you had a specific reason for adding that in this test.